ui/app/AppLayouts/Login/LoginFlow.qml

```diff
@@ -0,0 +1,249 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+import Qt.labs.settings 1.1
+
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+import StatusQ.Core.Theme 0.1
+
+import AppLayouts.Onboarding.enums 1.0
+import AppLayouts.Onboarding2.stores 1.0
+import AppLayouts.Onboarding2.controls 1.0
+import AppLayouts.Onboarding2.components 1.0
+
+import utils 1.0
+
+Page {
+    id: root
+
+    required property OnboardingStore onboardingStore
+
+    // [{keyUid:string, username:string, thumbnailImage:string, colorId:int, colorHash:var, order:int, keycardCreatedAccount:bool}]
+    required property var loginAccountsModel
+
+    property bool biometricsAvailable: Qt.platform.os === Constants.mac
+    required property bool isBiometricsLogin // FIXME should come from the loginAccountsModel for each profile separately?
```

Comment:
I just checked and indeed, it is per account because it comes from `localAccountSettings` and it changes depending on the selected profile
```nim
proc setSelectedLoginAccount*(self: Controller, keyUid: string, isKeycardAccount: bool) =
  ...
  let selectedAccount = self.getSelectedLoginAccount()
  singletonInstance.localAccountSettings.setFileName(selectedAccount.name)
```

Reply:
But wtf... how can this work when we are not logged in yet :thinking: 

Currently it's evaluated based on:
```qml
isBiometricsLogin: localAccountSettings.storeToKeychainValue === Constants.keychain.storedValue.store
```

`localAccountSettings`

Reply:
I mean, in the context of the login screen, this doesn't make sense, as we are not logged in yet

Reply:
This settings are not encryped. It's what you can find in `Status/qt`. `sensitive_settings` are the settings for when you are logged in

Reply:
> This settings are not encrypted. It's what you can find in `Status/qt`. `sensitive_settings` are the settings for when you are logged in

Ok, but how does the particular setting get associated with another profile (one that's not yet logged in)?